catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/model/RemoteStatus.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define(['backbone'],
+function (Backbone) {
+    var RemoteStatus = {};
+    RemoteStatus.Model = Backbone.Model.extend({
+        url: "/jolokia/exec/org.codice.ddf.ui.admin.api.ConfigurationAdmin:service=ui,version=2.3.0/getProperties/",
+            model: RemoteStatus.Model,
```

Comment:
I modeled this after the existing Status.js file. 
Do you mean it doesn't look right because I didn't put this into one of the other models (Service)? Or do you mean something else?